src/sentry_plugins/github/client.py

```diff
@@ -7,7 +7,7 @@
 from sentry_plugins.client import ApiClient, AuthApiClient
 
 
-class GitHubClientMixin(AuthApiClient):
+class GithubPluginClientMixin(AuthApiClient):
```

Comment:
Renamed the clients for the plugin to avoid confusion when auto-importing or searching the codebase.